site/en/setup.md

```diff
@@ -122,7 +131,36 @@ docker pull ocrd/all:maximum
 
 Replace `maximum` accordingly if you want the `minimum` or `medium` variant.
 
-(Also, if you want to keep the modules' git repos inside the Docker images – so you can keep making fast updates, without waiting for a new pre-built image but also without building an image yourself –, then add the suffix `-git` to the variant, e.g. `maximum-git`. This will behave like the native installation, only inside the container. Yes, you can also [commit changes](https://rollout.io/blog/using-docker-commit-to-create-and-change-an-image/) made in containers back to your local Docker image.)
+(Also, if you want to keep the modules' git repos inside the Docker images – so you can keep making 
```

Comment:
The explanation is still useful why it is useful to have the git repo inside the docker container. However, this has been the default for a while now. `maximum` is just an alias for `maximum-git` for backwards-compatibility.